MAUI/DataGrid/ToolTip.md

```diff
@@ -0,0 +1,210 @@
+---
+
+# ToolTip in MAUI DataGrid (SfDataGrid)
+
+The [SfDataGrid](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html) provides support for displaying tooltips. ToolTip provides the support to show the pop-up window that displays the information when interacting with cells of SfDataGrid. 
+
+To show tooltips:
+- **On Windows/Mac**: Hover the mouse cursor over any cell in the grid
+- **On Android/iOS**: Long press on any cell in the grid
+
+## Record cell tooltip
+
+To enable tooltip for datagrid, set the `SfDataGrid.ShowToolTip` property to `true`. This will display tooltip containing cell content when users interact with the cells.
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                              ItemsSource="{Binding OrdersInfo}"
+                              ShowToolTip="True" />
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.dataGrid.ShowToolTip = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="MAUI DataGrid displays ToolTip for Record Cell" src="Images\tooltip\maui-datagrid-tooltip-basic.png" width="404" />   
+
+You can enable tooltips for specific columns by setting the `DataGridColumn.ShowToolTip` property to `true` for the desired columns.
+
+{% tabs %}
+{% highlight XAML %}
+    <syncfusion:SfDataGrid.Columns>
+        <syncfusion:DataGridTextColumn HeaderText="Order ID" 
+                                       MappingName="OrderID" 
+                                       ShowToolTip="True"/>
+        <syncfusion:DataGridTextColumn HeaderText="Customer ID" 
+                                       MappingName="CustomerID" 
+                                       ShowToolTip="True"/>
+    </syncfusion:SfDataGrid.Columns>
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.Columns.Add(new DataGridTextColumn()
+{
+    HeaderText = "Order ID",
+    MappingName = "OrderID",
+    ShowToolTip = true
+});
+
+dataGrid.Columns.Add(new DataGridTextColumn()
+{
+    HeaderText = "Customer ID",
+    MappingName = "CustomerID",
+    ShowToolTip = true
+});
+{% endhighlight %}
+{% endtabs %}
+
+N> 
+The `DataGridColumn.ShowToolTip` property takes higher priority than the `SfDataGrid.ShowToolTip` property.
+
+## ToolTip Customization
+
+You can change the appearance of the ToolTip by customizing the style with TargetType as DataGridToolTipView.
+
+{% tabs %}
+{% highlight XAML %}
+    <ContentPage.Resources>
+        <Style TargetType="syncfusion:DataGridToolTipView">
+            <Setter Property="Background" Value="AliceBlue" />
+            <Setter Property="Stroke" Value="Red" />
+            <Setter Property="StrokeThickness" Value="2" />
+            <Setter Property="TextColor" Value="Brown" />
+        </Style>
+    </ContentPage.Resources>
+{% endhighlight %}
+{% endtabs %}
+<img alt="Customizing ToolTip Style in MAUI DataGrid" src="Images\tooltip\maui-datagrid-tooltip-style.png" width="404" /> 
+
+### Customize the ToolTip using ToolTipTemplate
```

Comment:
Introduce a topic Load views to the Tooltip and include following two as subtopics for the topic.

Customizing the ToolTip using DataTemplate
Customizing the ToolTip with DataTemplateSelector